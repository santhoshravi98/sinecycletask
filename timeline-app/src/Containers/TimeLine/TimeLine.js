import React, { Component } from 'react'
import Auxillary from '../../HOC/Auxillary/Auxillary'
import '../TimeLine/TimeLine.css'


class TimeLine extends Component {
    render() {
        return (
            <Auxillary>
                <ul className="list-timeline list-timeline-label" data-testid="ul-list">
                    {this.props.children}
                </ul>
            </Auxillary>
        )
    }
}



export default TimeLine;