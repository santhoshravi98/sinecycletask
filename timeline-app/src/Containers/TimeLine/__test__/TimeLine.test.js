import React from 'react'
import ReactDOM from 'react-dom'
import {render} from '@testing-library/react'
import TimeLine from '../TimeLine'
import '@testing-library/jest-dom'
import renderer from 'react-test-renderer'
import {fireEvent, cleanup} from '@testing-library/react';

afterEach(cleanup)
it("renders without crashing",()=>{
    const div = document.createElement("div");
    ReactDOM.render(<ul></ul>,div); 
})

// it("renders unordered list correctly",()=>{
//     const {getByTestId} = render(<TimeLine></TimeLine>)
//     expect(getByTestId("ul-list")).containsMatchingElement(<TimeLineItem />).toEqual(true)
// })

it("matches snapshot",() => {
const tree = renderer.create(<TimeLine></TimeLine>).toJSON();
expect(tree).toMatchSnapshot();
})

// it("has correct class",() => {
//     const  = render(<TimeLine></TimeLine>)
//     expect(getByTestId("ul-list").
//     })
