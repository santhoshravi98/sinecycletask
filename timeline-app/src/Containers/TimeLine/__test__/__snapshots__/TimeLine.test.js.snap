// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<ul
  className="list-timeline list-timeline-label"
  data-testid="ul-list"
/>
`;
